x-pack/test/security_solution_api_integration/package.json

```diff
@@ -63,7 +63,7 @@
     "nlp_cleanup_task:essentials:qa:serverless:release": "npm run run-tests:genai:basic_essentials nlp_cleanup_task serverless qaEnv",
 
     "entity_analytics:server:serverless": "npm run initialize-server:ea:trial_complete risk_engine serverless",
-    "entity_analytics:runner:serverless": "npm run run-tests:ea:trial_complete risk_engine serverless serverlessEnv",
+    "entity_analytics:runner:serverless": "npm run run-tests:ea:trial_complete risk_engine serverless serverlessEnv --",
```

Comment:
Is the -- at the end a leftover or are we good to keep it version control for general use?

Reply:
i guess we're keeping it 😅

Reply:
I am in full support ^^